integrationTests/vm/testInitializer.go

```diff
@@ -1009,18 +1015,30 @@ func CreatePreparedTxProcessorAndAccountsWithVMs(
 }
 
 func createMockGasScheduleNotifier() *mock.GasScheduleNotifierMock {
+	return createMockGasScheduleNotifierWithCustomGasSchedule(func(gasMap arwenConfig.GasScheduleMap) {})
```

Comment:
passing function pointers can be circumvented 
However, since we are in a testing framework we can live with that.